README.md

```diff
@@ -110,20 +110,43 @@ poetry run make protos
 poetry run hathor-cli run_node --status 8080
 ```
 
+It may take a considerable amount of time for it to sync all the transactions in the network. To speed things up, read below.
+
+#### Speeding up the sync
+You can use database snapshots to speed things up.
+
+We provide both testnet and mainnet snapshots. You can get the link to the latest snapshots this way:
+- Testnet: `curl https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest` 
```

Comment:
Should we add redirect urls? So we can redirect them accordingly when a new snapshot is available?

Reply:
The solution I proposed tries to solve this problem.

`https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest` is just a one-line file containing the real snapshot url.

That's why one have to do `wget $(curl https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest)` to get the snapshot

When a new snapshot is available, we just have to update the url in this file.

Reply:
Great solution! Thanks!